### PR TITLE
feat(mobile): iOS CD workflow

### PR DESCRIPTION
**Pull request by KuznetsovNikita**
iOS CD workflow

### CODE REVIEW

File: packages/mobile/package.json [hunk 1]
```diff
@@ -184,6 +184,7 @@
       "react-native-minimizer",
       "react-native-scrypt",
       "react-native-camera",
+      "react-native-pager-view",
```
Comment (KuznetsovNikita) [inline packages/mobile/package.json:187]: I have to move this dependency to package/mobile level for ios runner

Comment (bogoslavskiy) [reply packages/mobile/package.json:187]: You can use ^6.2.2, this fork outdated

Comment (KuznetsovNikita) [reply packages/mobile/package.json:186]: @bogoslavskiy 
Before that it was installed to `./node_modules/react-native-pager-view`
After this fix to `./packages/mobile/node_modules/react-native-pager-view`
Could you please check that it's work on your device?

**Comment by KuznetsovNikita** [comment]
@bogoslavskiy @sorokin0andrey 
What do you think if we will check-in `Podfile.lock` file?
Is it safety to install is from the scratch with knowing of pods version?

File: packages/mobile/package.json (hunk 1 above)
Comment (bogoslavskiy) [reply packages/mobile/package.json:186]: This is just symlink for metro, it will work

**Comment by bogoslavskiy** [comment]
> @bogoslavskiy @sorokin0andrey What do you think if we will check-in `Podfile.lock` file? Is it safety to install is from the scratch with knowing of pods version?

Yes, you can regenerate `Podfile.lock` from scratch, because all modules versions indicates in `yarn.lock`

**Review by sorokin0andrey — APPROVED** [review]

**Pull request merged**